src/components/pages/collectRecordFormPages/FishBeltForm/FishBeltForm.js

```diff
@@ -187,7 +202,7 @@ const FishBeltForm = ({ isNewRecord }) => {
         observationsDispatch({
           type: 'updateFishName',
           payload: {
-            observationId: newObservationToAdd,
+            observationId: observationIdToAddNewBenthicAttributeTo,
```

Comment:
k, I didnt want to spend too much time, but for fishName we are pretending its benthic attribute. Please rename this. (assuming you will take over branch and tidy up)